src/draw/nxp/pxp/lv_draw_pxp_blend.c

```diff
@@ -70,63 +66,60 @@
  *  STATIC PROTOTYPES
  **********************/
 
+static LV_ATTRIBUTE_MEM_ALIGN uint8_t temp_buf[PXP_TEMP_BUF_SIZE];
```

Comment:
This is a drastic measure to have a temporary buffer available, which not everyone is willing to do...

Reply:
Yes, but this buffer does not have to exist on SRAM but on external SDRAM.
It is a compromise if you want to do with PXP complex operations: like recolor + rotation.

In LVGL v9 I am thinking to remove it and let the PXP do only the simple tasks that is designed to.
E.G. It will do recolor and it will do rotation. But will not do both at the same time.